media-source/mediasource-is-type-supported.html

```diff
@@ -76,6 +82,17 @@
               'video/mp4;codecs="mp4a.40.2 , avc1.4d001e "',
               'video/mp4;codecs="avc1.4d001e,mp4a.40.5"',
           ], true, 'Test valid MP4 type');
+
+          mediasource_test(function(test, mediaElement, mediaSource)
```

Comment:
I understand, and that is an interesting datapoint. I assume that making Edge return "maybe" for canPlayType(... vorbis) might also not be a good idea, since apps may take that as a signal to attempt progressive playback of media including vorbis. I'll remove this MSE suite test, regardless, though in general, that non-normative note should be adhered to.